src/APIActionCreator.es6.js

```diff
@@ -98,7 +110,7 @@ export default class APIActionCreator extends ActionCreator {
 		);
 
 		invariant(
-			typeof description.method === 'string',
+			typeof method === 'string' || method,
```

Comment:
We actually do this check often for strings. 

`typeof description.route === 'string' || description.route,`

For example, which is 3 lines above this check.